src/pdn/src/via.cpp

```diff
@@ -2831,44 +2831,48 @@ void Via::writeToDb(odb::dbSWire* wire,
       = check_shapes(lower_, shapes.bottom);
   std::set<odb::dbSBox*> ripup_vias_top = check_shapes(upper_, shapes.top);
 
-  std::set<odb::dbSBox*> ripup_vias;
-  ripup_vias.insert(ripup_vias_bottom.begin(), ripup_vias_bottom.end());
-  ripup_vias.insert(ripup_vias_top.begin(), ripup_vias_top.end());
+  std::set<odb::dbSBox*> ripup_shapes;
+  ripup_shapes.insert(ripup_vias_bottom.begin(), ripup_vias_bottom.end());
+  ripup_shapes.insert(ripup_vias_top.begin(), ripup_vias_top.end());
```

Comment:
This makes me wonder if ripup_vias_bottom & ripup_vias_top should also be renamed.  I'll approve here and let you decide what makes sense.

Reply:
good point